client/me/help/help-contact-form/index.jsx

```diff
@@ -322,7 +322,7 @@ export class HelpContactForm extends React.PureComponent {
 			translate,
 			showingQandAStep,
 		} = this.props;
-		const hasQASuggestions = ! isEmpty( this.state.qanda );
+		const hasQASuggestions = isArray( this.state.qanda ) && ! isEmpty( this.state.qanda );
```

Comment:
From what I understand `this.state.qanda` should always be an array but sometimes we receive an error string and in the response in `#doQandASearch` we set `qanda` to that error string?

I think we should instead look at handling this error string differently, making `qanda`'s type strictly an `array`. This just makes for a reliable and representative piece of state and avoids having to do type checking every time we access it.

We could do that by doing something like so:
```js
...
.getQandA( query, site )
  .then( response =>
    if ( isArray( response ) ) {
      return this.setState( {
        qanda: response,
        // comments...
        sibylClicked: this.state.sibylClicked && areSameQuestions( this.state.qanda, qanda ),
      } )
    }
    // If response is not an array, empty the qanda array and set the error to the response value
    // note, we would need to handle clearing `qandaError` on successful calls.
    this.setState( { qanda: [], sibylClicked: false, qandaError:  response } )
  )
  .catch( () => this.setState( { qanda: [], sibylClicked: false } ) );
```